Directory.Build.targets

```diff
@@ -6,6 +6,7 @@
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
     <PackageReference Update="System.Text.Json" Version="4.6.0" />
     <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0" />
+    <PackageReference Update="Microsoft.Extensions.ObjectPool" Version="3.1.0" />
```

Comment:
What is it for?

Reply:
This was in order to try to recycle SqlQueryParser instances, instead of instantiating them for each command. I just ran another benchmark with and without, and the effect is inconclusive - pooling a relatively small object like this doesn't really make sense.

Also, if we actually get around to doing raw mode - where we don't parse the query - this step gets cut out completely, which is of course much better.

So I'm removing all this.